conf/machine/include/arch-riscv.inc

```diff
@@ -6,3 +6,5 @@ TUNE_ARCH = "${TUNE_ARCH_tune-${DEFAULTTUNE}}"
 TUNE_PKGARCH = "${TUNE_PKGARCH_tune-${DEFAULTTUNE}}"
 TUNE_CCARGS .= ""
 
+# QEMU usermode fails with invalid instruction error (YOCTO: #10304)
+MACHINE_FEATURES_BACKFILL_CONSIDERED_append = "${@bb.utils.contains('TUNE_FEATURES', '${DEFAULTTUNE}', ' qemu-usermode', '', d)}"
```

Comment:
this is as goog as 

`MACHINE_FEATURES_BACKFILL_CONSIDERED_append = "  qemu-usermode"`

secondly do we need this for risc64 ? or just risc32

Reply:
@kraj
Hmm change my PR to ->
```MACHINE_FEATURES_BACKFILL_CONSIDERED_append = " qemu-usermode"```
Ok I will test it. I look better and simple.

and my PR is for ```riscv32```, qemu usermode not work riscv32 I think
```riscv64``` build well without my PR.

So I suggest my PR like this. 
```MACHINE_FEATURES_BACKFILL_CONSIDERED_append_riscv32 = " qemu-usermode"```